lib/src/translateProvider.tsx

```diff
@@ -23,7 +24,8 @@ const TranslateProvider = (props: TranslateProviderProps) => {
   const { t, setLang, lang, isReady } = useTranslate(
     {
       root: props.root || defaultOptions.root,
-      lang: props.lang || defaultOptions.lang
+      lang: props.lang || defaultOptions.lang,
+      fallbackLang: props.lang || defaultOptions.fallbackLang
```

Comment:
Might be wrong reference, should it be like following?
```
fallbackLang: props.fallbackLang || defaultOptions.fallbackLang
```

Reply:
You're totally right, thanks